openfisca_uk/parameters/hmrc/tax_credits/working_tax_credit/elements/worker/rate.yaml

```
@@ -0,0 +1,29 @@
+description: The worker ("30-hour") element of Working Tax Credit.
```

I think in general we skip the "the"
```suggestion
description: Worker ("30-hour") element of Working Tax Credit.
```

---

Got it, yeah agreed.